src/aws.ts

```diff
@@ -361,4 +361,40 @@ export namespace $AWS {
       },
     });
   }
+
+  export namespace EventBridge {
+    /**
+     * @see https://docs.aws.amazon.com/eventbridge/latest/APIReference/API_PutEvents.html
+     */
+    export const putEvents = makeIntegration<
+      (
+        request: AWS.EventBridge.Types.PutEventsRequest
+      ) => AWS.EventBridge.Types.PutEventsResponse,
+      "EventBridge.putEvent"
+    >({
+      kind: "EventBridge.putEvent",
+      native: {
+        // Access needs to be granted manually
+        bootstrap: () => {},
+        /* istanbul ignore next */
+        preWarm: (prewarmContext: NativePreWarmContext) => {
+          /* istanbul ignore next */
```

Comment:
what is this instanbul thing?

Reply:
Trying to reduce the code coverage code injected into the function closures during tests, but it has not been successful. I'll probably just remove those. The code coverage code makes up a majority of the serialized functions. Mostly just a problem for us. I also though they may be contributing to issues I was having, but I don't think they are. Just another example of where we need control.